### PR TITLE
feat: add multi-step-form

### PR DESCRIPTION
**Pull request by MADAkwasi**

### CODE REVIEW

**Comment by fredrickyalley** [comment]
PR title

**Comment by fredrickyalley** [comment]
the button in the first form should be disabled if it does not take you to the next page

**Comment by fredrickyalley** [comment]
And also, while switching in between the pages, I can see the form being swiped to the left. I shouldn't see that

**Comment by MADAkwasi** [comment]
> PR title

done

**Comment by MADAkwasi** [comment]
> the button in the first form should be disabled if it does not take you to the next page

done

**Comment by MADAkwasi** [comment]
> And also, while switching in between the pages, I can see the form being swiped to the left. I shouldn't see that

removed

**Comment by fredrickyalley** [comment]
change the color when you disabled. Normally its greyed out to show that

**Comment by fredrickyalley** [comment]
The go back button should have a hover to show some reactiveness of the button

**Comment by fredrickyalley** [comment]
there is no validation on the name, email and number fields, You can type any character in all fields

**Comment by fredrickyalley** [comment]
I can continue from second and third form when i select something from there and proceed

**Comment by fredrickyalley** [comment]
When you are done with a form, It currently takes you back to home page. You should give the user the option to decide whether to go home or recreate a form.

**Comment by MADAkwasi** [comment]
> When you are done with a form, It currently takes you back to home page. You should give the user the option to decide whether to go home or recreate a form.

Screenshot (27)

The redirect feature was a requirement

**Comment by MADAkwasi** [comment]
> I can continue from second and third form when i select something from there and proceed

The third is not compulsory as per the requirements and the second form has its rate set to 'monthly' default

File: week_5/multi-step-form/src/app/components/heading/heading.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+@Component({
+  selector: 'app-heading',
+  standalone: true,
+  imports: [],
+  templateUrl: './heading.component.html',
+  styleUrl: './heading.component.css',
+})
+export class HeadingComponent {
+  @Input() headingTxt!: string;
```
Comment (fredrickyalley) [inline week_5/multi-step-form/src/app/components/heading/heading.component.ts:11]: Txt compared to Text is just missing e. You don't use shortform of words where the only difference in using the full word is just one character

Comment (MADAkwasi) [reply week_5/multi-step-form/src/app/components/heading/heading.component.ts]: Done

**Comment by MADAkwasi** [comment]
> there is no validation on the name, email and number fields, You can type any character in all fields

Done

**Comment by MADAkwasi** [comment]
> The go back button should have a hover to show some reactiveness of the button

Done

**Comment by MADAkwasi** [comment]
> I can continue from second and third form when i select something from there and proceed

corrected

**Comment by MADAkwasi** [comment]
> @MADAkwasi name field can receive numbers

done

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**